Makefile

```diff
@@ -51,23 +47,6 @@ isort:
 mypy:
 	pipenv run mypy lambdas
 
-### Developer Deploy Commands ###
-dist-dev: ## Build docker container (intended for developer-based manual build)
-	docker build --platform linux/amd64 \
-	    -t $(ECR_URL_DEV):latest \
-		-t $(ECR_URL_DEV):`git describe --always` \
-		-t $(ECR_NAME_DEV):latest .
-
-publish-dev: dist-dev ## Build, tag and push (intended for developer-based manual publish)
-	docker login -u AWS -p $$(aws ecr get-login-password --region us-east-1) $(ECR_URL_DEV)
-	docker push $(ECR_URL_DEV):latest
-	docker push $(ECR_URL_DEV):`git describe --always`
-
-update-lambda-dev: ## Updates the lambda with whatever is the most recent image in the ecr (intended for developer-based manual update)
-	aws lambda update-function-code \
-		--function-name $(FUNCTION_DEV) \
-		--image-uri $(ECR_URL_DEV):latest
-		
 ## Terraform-generated Makefile developer Deploy Commands ###
 dist-stage: ## Build docker container (intended for developer-based manual build)
```

Comment:
It looks like the dev and stage make commands both use the `-stage` suffix.

Reply:
Looking closer, I think they're identical? that's not what we want, right?

Reply:
hmm, good catch, maybe you pasted the stage commands twice @cabutlermit ?

Reply:
oh, dang! Let me go check.

Reply:
yup. bad copy/paste that is now fixed.